src/routes/quiz/[collection]/[id]/+page.svelte

```diff
@@ -330,25 +330,25 @@
                                 {#if currentQuizQuestion.image}
                                     <!-- svelte-ignore a11y-missing-attribute -->
                                     <img
-                                        class="quiz-question-image h-40"
```

Comment:
I saw that you removed h-40 from quiz-question-image.  That still needs to be there.  The android app hardcodes the height of the question image to 160px (h-40 equivalent).